versioned/storage/jdbc/src/main/java/org/projectnessie/versioned/storage/jdbc/AbstractJdbcPersist.java

```diff
@@ -508,6 +508,13 @@ private boolean[] upsertObjs(
               if (updated[i] == 1) {
                 r[batchIndexToObjIndex.get(i)] = true;
               }
+              if (updated[i] == PreparedStatement.SUCCESS_NO_INFO) {
```

Comment:
Since all special values are an error to Nessie, why not check `< 0`?

Reply:
To get a better error message.

Reply:
OK to change, better to cover all cases outside of 0 and 1.